tests/trustyai/conftest.py

```diff
@@ -179,3 +192,109 @@ def minio_data_connection(
         },
     ) as minio_secret:
         yield minio_secret
+
+
+@pytest.fixture(scope="class")
+def db_credentials(admin_client: DynamicClient, model_namespace: Namespace) -> Secret:
+    with Secret(
+        client=admin_client,
+        name="db-credentials",
+        namespace=model_namespace.name,
+        string_data={
+            "databaseKind": MARIADB,
+            "databaseName": "trustyai_database",
+            "databaseUsername": QUARKUS,
+            "databasePassword": QUARKUS,
+            "databaseService": MARIADB,
+            "databasePort": "3306",
+            "databaseGeneration": "update",
+        },
+    ) as db_credentials:
+        yield db_credentials
+
+
+@pytest.fixture(scope="session")
+def installed_mariadb_operator(admin_client: DynamicClient) -> None:
+    name = "mariadb-operator"
+    namespace = "openshift-operators"
+    install_operator(
+        admin_client=admin_client,
+        target_namespaces=[namespace],
+        name=name,
+        channel="alpha",
+        source="community-operators",
+        operator_namespace=namespace,
+        timeout=TIMEOUT_10_MIN,
+        install_plan_approval="Manual",
+    )
+    yield
+    uninstall_operator(admin_client=admin_client, name=name, operator_namespace=namespace, clean_up_namespace=False)
+
+
+@pytest.fixture(scope="session")
+def mariadb_operator_cr(admin_client: DynamicClient) -> MariadbOperator:
+    mariadb_csv = get_cluster_service_version(client=admin_client, prefix="mariadb", namespace=OPENSHIFT_OPERATORS)
+    alm_examples = mariadb_csv.get_alm_examples()
+    mariadb_operator_cr_dict = next(d for d in alm_examples if d["kind"] == "MariadbOperator")
```

Comment:
please avoid using one-letter vars so it is clear to someone looking at the code what is `d`